akka-remote/src/main/scala/akka/remote/RemoteActorRefProvider.scala

```diff
@@ -372,9 +372,6 @@ private[akka] class RemoteActorRefProvider(
     if (systemService) local.actorOf(system, props, supervisor, path, systemService, deploy, lookupDeploy, async)
     else {
 
-      if (!system.dispatchers.hasDispatcher(props.dispatcher))
-        throw new ConfigurationException(s"Dispatcher [${props.dispatcher}] not configured for path $path")
```

Comment:
That is done by LocalActorRefProvider. Delegating to LocalActorRefProvider for all cases but the
remote deployment case. For remote deployment there is another dispatcher config check.

Reply:
So this is the actual fix?

Reply:
yes, rest is the added tests